python/ray/tune/__init__.py

```diff
@@ -12,6 +14,9 @@
 from ray.tune.sample import (function, sample_from, uniform, choice, randint,
                              randn, loguniform)
 
+logger = logging.getLogger("ray.tune")
+logger.setLevel(logging.WARNING)
```

Comment:
is this kosher? there's a global ray setting for warnings too I think. I'm not quite sure what are the best user practices for this.

Reply:
Oh actually just copied this from the fork, not sure what the change is for I can get rid of it.